TestON/RobotTests/README

```diff
@@ -0,0 +1,16 @@
+This is a scale test find the max number of switches that can connect to one ONOS Controller. The test script does not create a cluster of nodes, but that can be easily implemented.
+
+Pre-conditions to running this test is Robot Framework + external robot libraries:
+
+pip install pybot
```

Comment:
I don't think this is needed. The only PyBot project I found with pip search was an irc bot and it was an externally hosted module. The pybot command seems to get installed when the robotframework modules are installed.